.github/workflows/formatter.yml

```diff
@@ -0,0 +1,34 @@
+name: Code Formatter
+
+on:
+  push:
```

Comment:
I guess this one can run only for PRs.

Reply:
Fair point.

Out of curiosity, though, do you know if somebody can push into `master` directly?

Reply:
No, it's not allowed.